src/Sarif.Multitool/MultitoolResources.resx

```diff
@@ -129,6 +129,21 @@
   <data name="ErrorNoGroupInFileName" xml:space="preserve">
     <value>Error: No "group" (the part of the file name before the dash) found in '{0}'. All file names must be of the form [Group]-[RunID] and each adjacent pair in the same group is compared.</value>
   </data>
+  <data name="ErrorUnknownFilingTargetType" xml:space="preserve">
```

Comment:
>ErrorUnknownFilingTargetType [](start = 14, length = 28)

since this is a common file for all potential "multitool" related messages, we should append the ones specific bug filing component with some special keyword e.g. "FileWorkItems_"


(Ryley says)